vignettes/covariance_fixed_effect.Rmd

```diff
@@ -0,0 +1,85 @@
+---
+title: "Covariance of Fixed Effects in `mmrm`"
```

Comment:
```suggestion
title: "Coefficients Covariance Matrix Adjustment in `mmrm`"
```
and similar below. (Reason: Because we don't have any random effects, we don't talk about fixed effects in the package)